packages/@aws-cdk/aws-scheduler-alpha/README.md

```diff
@@ -97,7 +98,31 @@ const oneTimeSchedule = new Schedule(this, 'Schedule', {
 
 ### Grouping Schedules
 
-TODO: Group is not yet implemented. See section in [L2 Event Bridge Scheduler RFC](https://github.com/aws/aws-cdk-rfcs/blob/master/text/0474-event-bridge-scheduler-l2.md)
+Your AWS account comes with a default scheduler group. You can access default group in CDK with:
+
+```text
```

Comment:
these examples can be in `ts` now? Since they should be fully implemented right. Then you will also have to double check with `yarn rosetta:extract --strict` or else the PR build may fail :)

Reply:
Updated now, good catch